packages/zoe/src/contracts/oracle.js

```diff
@@ -0,0 +1,145 @@
+// @ts-check
+import { assert, details } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+import { trade } from '../contractSupport';
+
+import '../../exported';
+
+/**
+ * This contract provides oracle queries for a fee or for pay.
+ *
+ * @type {ContractStartFn}
+ *
+ */
+const start = async zcf => {
+  const { oracleDescription } = zcf.getTerms();
+
+  const {
+    brands: { Fee: feeBrand },
+    maths: { Fee: feeMath },
+  } = zcf.getTerms();
+
+  /** @type {OracleHandler} */
+  let handler;
+  /** @type {string} */
+  const description = oracleDescription;
+
+  const { zcfSeat: feeSeat } = zcf.makeEmptySeatKit();
+
+  let lastIssuerNonce = 0;
+  /** @type {string} */
+  let revoked;
+
+  /** @type {OracleCreatorFacet} */
+  const realCreatorFacet = {
+    async addFeeIssuer(issuerP) {
+      lastIssuerNonce += 1;
+      const keyword = `OracleFee${lastIssuerNonce}`;
+      await zcf.saveIssuer(issuerP, keyword);
+    },
```

Comment:
This is not used in this contract, and we should remove it because it gives people the impression that multiple fee issuers are accepted, but they are not.